packages/desktop-client/src/components/transactions/MobileTransaction.js

```diff
@@ -99,6 +177,789 @@ function Status({ status }) {
+                  bare={true}
+                >
+                  <SvgTrash
+                    width={17}
+                    height={17}
+                    style={{ color: colors.r4 }}
+                  />
+                  <Text
+                    style={{
+                      color: colors.r4,
+                      marginLeft: 5,
+                      userSelect: 'none',
+                    }}
+                  >
+                    Delete transaction
+                  </Text>
+                </Button>
+              </View>
+            )}
+          </View>
+
+          <View
+            style={{
+              paddingLeft: EDITING_PADDING,
+              paddingRight: EDITING_PADDING,
+              paddingTop: 15,
+              paddingBottom: 15,
+              backgroundColor: colors.n11,
+              borderTopWidth: 1,
+              borderColor: colors.n10,
+              marginTop: 'auto',
+            }}
+          >
+            {adding ? (
+              <Button onClick={() => this.onAdd()}>
+                <SvgAdd width={17} height={17} style={{ color: colors.b3 }} />
+                <Text
+                  style={[styles.text, { color: colors.b3, marginLeft: 5 }]}
+                >
+                  Add transaction
+                </Text>
+              </Button>
+            ) : (
+              <Button onClick={() => this.onSave()}>
+                <SvgPencilWriteAlternate
+                  style={{ width: 16, height: 16, color: colors.n1 }}
+                />
+                <Text
+                  style={[styles.text, { marginLeft: 6, color: colors.n1 }]}
+                >
+                  Save changes
+                </Text>
+              </Button>
+            )}
+          </View>
+
+          {/* <ExitTransition
+            alive={editingChild}
+            withProps={{
+              transaction:
+                editingChild && transactions.find(t => t.id === editingChild),
+            }}
+          > */}
+          {renderChildEdit({
+            transaction:
+              editingChild && transactions.find(t => t.id === editingChild),
+            amountSign: forcedSign,
+            getCategoryName: id => (id ? lookupName(categories, id) : null),
+            navigation: navigation,
+            onEdit: this.onEdit,
+            onStartClose: this.onSaveChild,
+          })}
+          {/* </ExitTransition> */}
+        </View>
+      </View>
+      // </KeyboardAvoidingView>
+    );
+  }
+}
+
+function isTemporary(transaction) {
+  return transaction.id.indexOf('temp') === 0;
+}
+
+function makeTemporaryTransactions(currentAccountId, lastDate) {
+  return [
+    {
+      id: 'temp',
+      date: lastDate || monthUtils.currentDay(),
+      account: currentAccountId,
+      amount: 0,
+      cleared: false,
+    },
+  ];
+}
+
+function TransactionEditUnconnected(props) {
+  // componentDidMount() {
+  //   this.props.getCategories();
+  //   this.props.getAccounts();
+  // }
```

Comment:
Can this be removed?

Reply:
Yup, done via https://github.com/actualbudget/actual/pull/1340/commits/827802d4ddc29e4117cd37a4494e35ba74069115